base/src/com/google/idea/blaze/base/command/mod/BlazeModRunnerImpl.java

```diff
@@ -38,6 +38,9 @@
 
 public class BlazeModRunnerImpl extends BlazeModRunner {
 
+  private static final String DUMP_REPO_MAPPING = "dump_repo_mapping";
+  private static final String ROOT_WORKSPACE = "";
```

Comment:
previously it was "workspace", not an empty string

Reply:
I see the context there: https://github.com/bazelbuild/intellij/issues/6664#issuecomment-2329269203

Could you summarize that in a code comment?